### PR TITLE
[conductor] update console link

### PR DESCRIPTION
**Pull request by itsjustkevin**
Update console links to point to packaging and building dashboards. 

continuation of closed pr https://github.com/flutter/flutter/pull/117360

### CODE REVIEW

File: dev/conductor/core/lib/src/state.dart [hunk 1]
```diff
@@ -45,6 +45,9 @@ String luciConsoleLink(String channel, String groupName) {
   );
   final String consoleName =
       channel == 'master' ? groupName : '${channel}_$groupName';
+  if (groupName == 'packaging') {
```
Comment (CaseyHillers) [inline dev/conductor/core/lib/src/state.dart:48]: lgtm, tests need an update

**Review by XilaiZhang — APPROVED** [review]
images (1)

**Comment by itsjustkevin** [comment]
@CaseyHillers and @XilaiZhang I am stuck in google testing for this PR.  Any advice?

**Comment by CaseyHillers** [comment]
> @CaseyHillers and @XilaiZhang I am stuck in google testing for this PR. Any advice?

This PR is ~300 commits behind tip of tree. Can you rebase it? `git fetch --all && git rebase upstream/master && git push origin HEAD -f`

**Pull request merged**